apis/compute/v1alpha2/types.go

```diff
@@ -34,12 +40,53 @@ const (
 	DefaultNodeCount = 1
 )
 
+// Error strings
+const (
+	errResourceIsNotAKSCluster = "The managed resource is not a AKSCluster"
```

Comment:
```suggestion
	errResourceIsNotAKSCluster = "The managed resource is not an AKSCluster"
```

Reply:
While we're at it, error strings should not start with capitals in Go.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
fixed these in the referencing #39